samples/spring-security-xsuaa-usage/src/main/java/sample/spring/xsuaa/TestController.java

```diff
@@ -153,6 +154,7 @@ public String requestClientCredentialsToken() throws TokenFlowException {
     public String requestUserToken(@AuthenticationPrincipal Jwt jwt) throws TokenFlowException {
```

Comment:
Hi @amritanshvns 
thanks for the PR! I would change the Argument to `@AuthenticationPrincipal Token token`... in that case a conversion is not needed.
Thanks and best regards,
Nena